field/time/classes/item.php

```diff
@@ -259,7 +259,12 @@ public function item_add_mandatory_plugin_fields(&$record) {
      * @return int unixtime
      */
     public function item_time_to_unix_time($hour, $minute) {
-        return (mktime($hour, $minute, 0, SURVEYPROFIELD_TIME_MONTHOFFSET, SURVEYPROFIELD_TIME_DAYOFFSET, SURVEYPROFIELD_TIME_YEAROFFSET));
+        $unixtime = mktime(
+                        $hour, $minute, 0,
+                        SURVEYPROFIELD_TIME_MONTHOFFSET, SURVEYPROFIELD_TIME_DAYOFFSET, SURVEYPROFIELD_TIME_YEAROFFSET
+                    );
```

Comment:
While surely codechecker doesn't report this... the closing parenthesis line should be aligned with `$unixtime` not with `mktime(`.

Anyway, it's not critical. In both cases it's readable that is the main point.